### ISSUE TITLE
Can not access remote container after several days

### ISSUE THREAD

**Issue by Dilshat** [subutai-io/p2p#151]
I created environment two days ago, the container on remote peer (not in LAN, over WAN) is not accessible via p2p. It is running and accessible via SS.
[p2p.txt](https://github.com/subutai-io/p2p/files/315530/p2p.txt)

**Comment by Dilshat** [comment]
ping between containers works
just can not access it via tunnel
may be related to https://github.com/subutai-io/tray/issues/65

**Comment by tzununbekov** [comment]
Got environment for 3 days now, RHs and network was intentionally interrupted several times, ssh still works

> just can not access it via tunnel

could you please explain what does it mean?

**Comment by Dilshat** [comment]
@tzununbekov I create ssh tunnel via SS UI. It works on locally hosted containers and does not on remote. When i do ping from one container to another it works.
Also Peers are not within the same LAN.

**Comment by tzununbekov** [comment]
@Dilshat hm, I'm confused now. Have you created tunnel 2 days ago? Or now newly created tunnel doesn't work? Is it a reproducible scenario? If not, can I get an access to problematic host to investigate the problem?

**Comment by tzununbekov** [comment]
btw, maybe this is related?
https://github.com/subutai-io/base/issues/1082

**Comment by Dilshat** [comment]
@tzununbekov tunnels is fresh, unfortunately i deleted the env, will try to reproduce

**Comment by Dilshat** [comment]
Does not reproduce on last versions

**Comment by asanaliev** [comment]
About several hours ago I created environment between two peers: 
![image](https://cloud.githubusercontent.com/assets/7465404/16475571/6eff3198-3ea1-11e6-9d50-414752f2b490.png)

At the beginning connection between peers was stable: both ping and ssh. But now ping doesn't work for the containers. To test, I created another environment between same peers:

![image](https://cloud.githubusercontent.com/assets/7465404/16475618/caa07d54-3ea1-11e6-8e07-13191e07b688.png)

Here, ping works. Seems, p2p has some issues to keep connection between peers for long time.
